ci-operator/step-registry/stackrox/install-helm/stackrox-install-helm-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: stackrox-install-helm
+  from_image:
```

Comment:
Look if we can build an image with `helm` and persist or use another image that already contains `jq`, `helm`, etc.

Reply:
We will leave this for future improvements.